packages/neuron-wallet/src/startup/sync-block-task/sync.ts

```diff
@@ -44,19 +44,18 @@ export const switchNetwork = async (url: string, genesisBlockHash: string, chain
   // listen to address created
   addressCreatedSubject.subscribe(async (addresses: Address[]) => {
     if (blockListener) {
-      const infos: LockHashInfo[] = (await Promise.all(
-        addresses.map(async addr => {
-          const hashes: string[] = await LockUtils.addressToAllLockHashes(addr.address)
-          // undefined means false
-          const isImporting: boolean = addr.isImporting === true
-          return hashes.map(h => {
-            return {
-              lockHash: h,
-              isImporting,
-            }
-          })
+      const lockUtils = new LockUtils(await LockUtils.loadSystemScript(url))
```

Comment:
These parts look like the same as https://github.com/nervosnetwork/neuron/pull/1011/files#diff-1449216a4a57add5c2a9df562c148952R45-R56

Reply:
Duplicated but not a big issue. `sync` is for non-indexer module and `indexer` is for indexer module. I'd say it's ok to have them not reusable to isolate the implementations.

BTW reusing code or logic is not concern of this PR.